frappe/public/js/frappe/form/controls/attach.js

```diff
@@ -79,10 +79,12 @@ frappe.ui.form.ControlAttach = frappe.ui.form.ControlData.extend({
 		this.value = value;
 		if(this.value) {
 			this.$input.toggle(false);
-			if(this.value.indexOf(",")!==-1) {
-				var parts = this.value.split(",");
-				var filename = parts[0];
-				dataurl = parts[1];
+			// value can also be using this format: FILENAME,DATA_URL
+			// Important: We have to be careful because normal filenames may also contain ","
+			var twoPartMatches = this.value.match(/^([^:]+),(.+):(.+)$/);
```

Comment:
Please use snake-case. `two_part_matches` and not `twoPartMatches`.

Also, use `let` instead of `var` for consistency.

Reply:
@netchampfaris I've changed it as your request.